src/extension.ts

```diff
@@ -368,3 +386,36 @@ function handleLogViaKamelCli(integrationName: string) : Promise<string> {
 export function getStashedContext() : vscode.ExtensionContext {
 	return stashedContext;
 }
+
+function removeIntegrationLogView(integrationName: string) : Promise<string> {
+	return new Promise<string>( async () => {
+		if (closeLogViewWhenIntegrationRemoved) {
+			LogsPanel.currentPanels.forEach((value : LogsPanel) => {
+				var title = value.getTitle();
+				if (title.indexOf(integrationName) > 0) {
+					value.disposeView();
+				}
+			});
+		} else {
+			LogsPanel.currentPanels.forEach((value : LogsPanel) => {
+				var title = value.getTitle();
+				if (title.indexOf(integrationName) > 0) {
+					updateLogViewTitleToStopped(value, title);
+				}
+			});
+		}
+	});
+}
+
+function updateLogViewTitleToStopped(panel: LogsPanel, title: string) {
+	if (panel && title) {
+		// make sure we only show the log as stopped once 
+		const stoppedString = ` [Integration stopped]`;
+		let boolHasIntegrationStopped = title.indexOf(stoppedString) > 0;
```

Comment:
might be better to check >= 0, in case the "[Integration stopped]" is put at te beginning of the title one day, and check without the first space in this case